### ISSUE TITLE
Unsuccessful installation on Ubuntu 20.04 RX 580

### ISSUE THREAD

**Issue by facorazza** [RadeonOpenCompute/ROCm#1412]
Followed the installation process: https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html#ubuntu
When I run:
```
/opt/rocm/bin/rocminfo
/opt/rocm/opencl/bin/clinfo
```
I get:
```
zsh: no such file or directory: /opt/rocm/bin/rocminfo
dlerror: libamd_comgr.so.1: cannot open shared object file: No such file or directory
ERROR: clGetPlatformIDs(-1001)
```

Kernel: 5.4.0-42-generic

**Comment by xuhuisheng** [comment]
Seems some deb didn't install correctly. libamd_comgr.so should be in /opt/rocm/lib/.

BTW: AMD drop RX580 offical support on ROCm-4.0.
https://github.com/xuhuisheng/rocm-build/tree/develop/gfx803

**Comment by facorazza** [comment]
Well that's unfortunate

**Comment by facorazza** [comment]
Does that mean that the installation won't work at all or that it might work but it's not guaranteed?

**Comment by xuhuisheng** [comment]
Actually, we can install ROCm-4.0 successfully. But there are some NaN loss situation when running tensorflow/pytorch.

**Comment by facorazza** [comment]
I'm actually planning on using openmp but some tf would be nice. Should I try to install it again and check for errors before rebooting?

**Comment by ROCmSupport** [comment]
Hi @facorazza 
Thanks for reaching out.
We are not officially supporting gfx8 devices now.
But some things might work. In your case, looks like installation did not go well.
Request you to clean the system from rocm and reinstall freshly.
Thank you.
